contents/blog/posthog-changelog.md

```diff
@@ -17,7 +17,28 @@ featuredImageType: full
 
 Every Friday we update this page with the latest new features, fixes, and updates on PostHog Cloud in the last seven days. We also feature notable additions to the blog and docs. 
 
-> Some new features you read about below may still be in beta, behind a feature flag, or only available to paying users. Want to see which betas are currently available? [Check the roadmap](/roadmap)!
+> Some new features you read about below may still be in beta, behind a feature flag, or only available to paying users. Want to see which betas are currently available? [Check the roadmap](/roadmap). We also [welcome your feedback](http://app.posthog.com/home#supportModal) on all new features! 
+
+## May 12, 2023
+
+#### Beta: Early access feature management
+![feature management](../images/blog/array/feature_management.mp4)
+
+We heard you like betas, so here's a beta for your betas. 
+
+Early access feature management got it's start at [our recent offsite hackathon](/blog/aruba-hackathon) and was intended to make it easier for us to accept users into our beta process, but we're also rolling it out as a feature for you to take advantage of in your own products. Leveraging feature flags, it enables you to present an early access menu to your users so that they can self opt-in or out of betas and other early access features. In the image above you can see it in action as we demo a user opting in to a beta of our new, upcoming UI refresh. 
+
+Feature management is still very much in beta, so we'd [love feedback about it](http://app.posthog.com/home#supportModal) - and you can [find out more in the docs](/docs/feature-flags/early-access-feature-management). 
+
+#### Idle session detection
+We've improved the detection logic for session replays so that we'll no longer capture sessions if the user is idle for 300 seconds (5 minutes) or longer. The idea is that you'll capture fewer sessions where the user isn't actually doing anything, though it _may_ also mean we trigger resets of the session more often. 
+
+Basically, its a balancing act and [we'd welcome your feedback about it](http://app.posthog.com/home#supportModal) if you start to notice sessions ending prematurely, or a substantial increase in the number of recordings. 
+
+#### Funnels by sessions
+![funnels by sessions](../images/blog/array/funnels_session.mp4)
+
+We've added a new way to aggregate the results of a funnel: by session. One possible use-case for this would be comparing funnel performance when aggregated by unique users and sessions, to understand if your users generally require multiple session to convert, or to get a feel for how 'sticky' your content is. 
```

Comment:
```suggestion
We've added a new way to aggregate the results of a funnel: by session. One possible use-case for this would be comparing funnel performance when aggregated by unique users and sessions, to understand if your users generally require multiple sessions to convert, or to get a feel for how 'sticky' your content is. 
```